library/src/blas2/rocblas_sbmv.hpp

```diff
@@ -139,7 +182,8 @@ rocblas_status rocblas_sbmv_template(rocblas_handle handle,
     }
     else
     {
-        if(!*alpha && *beta == 1)
+        // quick return only for non-batched
```

Comment:
Seems good.
Alternatively, if we're always passing in stride_alpha = stride_beta = 0 from the C API, you could do a check like:
`if(stride_alpha == 0 && stride_beta == 0 && !*alpha && *beta == 1)`
This way even if batch_count != 1 we can still quick return if we're always using the quick return alpha/beta.

Reply:
For batched non-strided you can't rely on strides == 0 test